spec/controllers/hyrax/collections_controller_spec.rb

```diff
@@ -31,7 +31,7 @@
       end
 
       it "returns the collection and its members", :with_nested_reindexing do # rubocop:disable RSpec/ExampleLength
-        expect(controller).to receive(:add_breadcrumb).with(I18n.t('hyrax.dashboard.title'), Hyrax::Engine.routes.url_helpers.dashboard_path(locale: 'en'))
+        expect(controller).to receive(:add_breadcrumb).with(I18n.t('hyrax.controls.home'), Hyrax::Engine.routes.url_helpers.root_path(locale: 'en'))
```

Comment:
You changed this test. But you didn't change `CollectionsController#show` to have this behavior.